oops/polymorphism/virtual.cpp

```diff
@@ -0,0 +1,29 @@
+#include<iostream>
+using namespace std;
+
+class Base{
+    public:
+    int x;
+    virtual void display(){             //if we add virtual key word then on run time the obj is bind with the class obj
```

Comment:
```suggestion
    virtual void display(){             
    //if we add virtual key word then on run time the obj is bind with the class obj
```
try to informative comment in the new line.